crates/bazed/src/main.rs

```diff
@@ -34,7 +34,10 @@ async fn main() -> Result<()> {
     });
 
     if run_frontend {
+        #[cfg(feature = "tauri")]
         bazed_tauri::run_frontend();
+        #[cfg(not(feature = "tauri"))]
+        eprintln!("please compile with the `tauri`-feature enabled or pass --no-frontend");
```

Comment:
panic! would be better, no?

Reply:
would it be? I don't really see why we should panic here if we don't have to :/

Reply:
We also can't do what the user wants in any way.

Reply:
I guess? Still not a fan, but lemme just panic